src/test/java/org/sonarsource/sonarlint/ls/connected/ProjectBindingManagerTests.java

```diff
@@ -258,56 +224,14 @@ void get_binding_for_single_file_uses_parent_dir_as_basedir() {
     verify(fakeEngine).calculatePathPrefixes(eq(PROJECT_KEY), argThat(set -> set.contains(FILE_PHP)));
   }
 
-  @Test
-  void get_binding_should_update_if_global_storage_missing() {
-    mockFileInABoundWorkspaceFolder();
-
-    when(fakeEngine.getGlobalStorageStatus()).thenReturn(null, globalStorageStatus);
-
-    var binding = underTest.getBinding(fileInAWorkspaceFolderPath.toUri());
-    assertThat(binding).isNotEmpty();
-
-    verify(fakeEngine, times(1)).update(any(), any(), any());
-    verify(fakeEngine, never()).updateProject(any(), any(), any(), anyBoolean(), any(), any());
-  }
-
-  @Test
-  void get_binding_should_update_if_global_storage_is_stale() {
-    mockFileInABoundWorkspaceFolder();
-
-    when(globalStorageStatus.isStale()).thenReturn(true, false);
-
-    var binding = underTest.getBinding(fileInAWorkspaceFolderPath.toUri());
-    assertThat(binding).isNotEmpty();
-
-    verify(fakeEngine, times(1)).update(any(), any(), any());
-    verify(fakeEngine, never()).updateProject(any(), any(), any(), anyBoolean(), any(), any());
-  }
-
   @Test
   void get_binding_should_update_if_project_storage_missing() {
     mockFileInABoundWorkspaceFolder();
 
-    when(fakeEngine.getProjectStorageStatus(PROJECT_KEY)).thenReturn(null);
-
-    var binding = underTest.getBinding(fileInAWorkspaceFolderPath.toUri());
-    assertThat(binding).isNotEmpty();
-
-    verify(fakeEngine, never()).update(any(), any(), any());
-    verify(fakeEngine, times(1)).updateProject(any(), any(), eq(PROJECT_KEY), anyBoolean(), any(), any());
-  }
-
-  @Test
-  void get_binding_should_update_if_project_storage_stale() {
-    mockFileInABoundWorkspaceFolder();
-
-    when(projectStorageStatus.isStale()).thenReturn(true);
-
     var binding = underTest.getBinding(fileInAWorkspaceFolderPath.toUri());
     assertThat(binding).isNotEmpty();
 
-    verify(fakeEngine, never()).update(any(), any(), any());
-    verify(fakeEngine, times(1)).updateProject(any(), any(), eq(PROJECT_KEY), anyBoolean(), any(), any());
+    verify(fakeEngine, times(1)).updateProject(any(), any(), eq(PROJECT_KEY), any());
```

Comment:
I think `times(1)` is the default value but I might be wrong. It appears in a lot of places by the way